src/core/grid_based_algorithms/lbgpu_cuda.cu

```diff
@@ -2646,36 +2521,14 @@ __device__ void calc_viscous_force(
 #endif
 
     /** add stochastic force of zero mean (Ahlrichs, Duenweg equ. 15)*/
-#ifdef FLATNOISE
-    random_01(rn_part);
-    viscforce_density[0 + ii * 3] +=
-        para->lb_coupl_pref[ii] * (rn_part->randomnr[0] - 0.5f);
-    viscforce_density[1 + ii * 3] +=
-        para->lb_coupl_pref[ii] * (rn_part->randomnr[1] - 0.5f);
-    random_01(rn_part);
-    viscforce_density[2 + ii * 3] +=
-        para->lb_coupl_pref[ii] * (rn_part->randomnr[0] - 0.5f);
-#elif defined(GAUSSRANDOMCUT)
-    gaussian_random_cut(rn_part);
+    float4 random_floats =
+        random_wrapper_philox(part_index, ii, philox_counter);
```

Comment:
Right now this generates the same random numbers as for some fluid cells. You should replace `ii` with `ii + 1024` or something.